### ISSUE TITLE
Key binding doesn't work with ctrl+tab key

### ISSUE THREAD

**Issue by ronaldsuwandi** [marta-file-manager/marta-issues#184]
Tried adding `ctrl+tab` and `ctrl+shift+tab` key to activate next/previous tab with no luck (works with other key though)

```
{
    "keyBindings": {
        "core.activate.next.tab": ["Alt+Cmd+Right", "Ctrl+Tab"],
        "core.activate.previous.tab": ["Alt+Cmd+Left", "Ctrl+Shift+Tab"]
    }
}
```

**Comment by yanex** [comment]
Looks like macOS swallows Ctrl+Tab, and the key event is just not passed to Marta.
Do you know any native macOS applications which allow to set `Ctrl+Tab` as a hotkey?

**Comment by warpkanal** [comment]
Not sure whether these allow to re-customize, but a lot of apps use `Ctrl+Tab` to switch tabs, e.g. Chrome, Intellij, NimbleCommander (the latter should qualify as "native" macOS app, although I don't think that's important), so I guess it's not generally swallowed.
Maybe you have a `Ctrl+Tab` binding in macOS-System Preferences-Keyboard shortcuts, and maybe only then it's swallowed?

**Comment by sfieten** [comment]
This issue seems to be back in version 0.6 as my custom keybinding with `Ctrl+Tab` for next tab now switches panes and `Shift+Ctrl+Tab` does nothing.

**Comment by yanex** [comment]
Thanks for the report! I will investigate this shortly.

**Comment by sfieten** [comment]
I am sorry to report, but this still doesn't work in version 0.6.1

**Comment by fabeit** [comment]
It seems I cannot remap these keybindings, I have tried 

keyBindings {
    "Control+Tab" { id "core.activate.next.tab" }
    "Cmd+Tab+Alt" { id "core.activate.previous.tab" }
    
}

but doesn't work.

**Comment by yanex** [comment]
Will be fixed in 0.8.1 RC4.
